CONTRIBUTING.md

```diff
@@ -64,7 +79,7 @@ You should submit your patch as a git branch named after the Github issue, such
 as `#3`\. This is called a _topic branch_ and allows users to associate a
 branch of code with the issue.
 
-It is a best practice to have your commit message have a _summary line_ that
+It is the best practice to have your commit message have a _summary line_ that
 includes the issue number, followed by an empty line and then a brief
```

Comment:
Should we enforce always having an issue first?

Reply:
I don't think it's that useful, in fact PR are already some kind of issues, and probably enough for now :)

Reply:
Even GitHub is joining them under https://github.com/EGI-Foundation/documentation/issues?q=is%3Aopen+